<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncoiceDetectorUI</class>
 <widget class="QMainWindow" name="IncoiceDetectorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InvoiceDetect</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_Workspace">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_choose_file">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: undefined;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_choose_Folder">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Folder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>Workspace</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Detect">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>411</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="result_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(245, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>791</width>
      <height>81</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(校标-校徽.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>:/images/InvoiceDetector2.0/png/test/校标-校徽.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_co">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Carlito&quot;;</string>
     </property>
     <property name="text">
      <string>Created by JAKA,Co,2024</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>TrueSunset.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 290 16pt &quot;Microsoft YaHei&quot;;</string>
     </property>
     <property name="text">
      <string>InvoiceDetector 2.1</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_logo</zorder>
    <zorder>line_title</zorder>
    <zorder>label_co</zorder>
    <zorder>label_3</zorder>
   </widget>
   <widget class="QFrame" name="frame_Searching">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_Searching">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>Opearting</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_code">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>发票代码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_invoice_info">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mod">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_del">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_INFO">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>20</width>
       <height>371</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_invoicecode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_date">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_buyername">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_buyercode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_sellername">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_sellercode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_totalprice">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_code_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>发票代码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Searching_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Consolas&quot;;</string>
     </property>
     <property name="text">
      <string>Searching Results</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_date">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>开票日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_buyername">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>购方名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_buyercode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>购方编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_sellername">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>销方名称：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_sellercode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>销方编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_totalprice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>价税合计：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_invoice_notes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_notes">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_PIC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(BG.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Author">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_De">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Carlito&quot;;</string>
     </property>
     <property name="text">
      <string>Designed by</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Carlito&quot;;</string>
     </property>
     <property name="text">
      <string>Racheus Zhao,Shuo Qi,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Copy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 9pt &quot;Carlito&quot;;</string>
     </property>
     <property name="text">
      <string>JAKA Intern.CopyRight©2024</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_JAKA">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(logo.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>版本(V)</string>
    </property>
    <addaction name="actionVersion2_0_qt"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVersion2_0_qt">
   <property name="text">
    <string>Version2.0 (qt)</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../sjtu.qrc"/>
 </resources>
 <connections/>
</ui>
